src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -406,18 +402,21 @@ Status TableEntryToFlatbuf(RedisModuleKey *table_key, RedisModuleString *entry_i
                                        fbb.CreateVector(&data, 1));
     fbb.Finish(message);
   } break;
-  case REDISMODULE_KEYTYPE_ZSET: {
+  case REDISMODULE_KEYTYPE_LIST: {
+    RedisModule_CloseKey(table_key);
+    RedisModuleString *table_key_str = PrefixedKeyString(ctx, prefix_str, entry_id);
+    RedisModuleCallReply *reply =
+        RedisModule_Call(ctx, "LRANGE", "sll", table_key_str, 0, -1);
```

Comment:
This is the line that you think might be slow? Maybe add a TODO to replace this once Redis modules have support for low-level list iteration?